idx/idx.go

```diff
@@ -147,6 +147,17 @@ type MetricIndex interface {
 	// LastUpdate time is >= the given value.
 	Tags(int, string, int64) ([]string, error)
 
+	// AutoCompleteTags is used to generate a list of possible values that could
+	// complete a given prefix. It also accepts additional conditions to further
```

Comment:
"values that could complete a given prefix" this is confusing because AFAICT we're not talking about tag values, but keys.

Reply:
also, what kind of conditions? expressions such as those accepted by FindByTag ?

Reply:
updated: https://github.com/grafana/metrictank/pull/779/commits/9d217cd83c383b8046976e88be2a0f6c43946d2f